jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/skaffold/SyncMapTaskTest.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle.skaffold;
+
+import com.google.cloud.tools.jib.api.AbsoluteUnixPath;
+import com.google.cloud.tools.jib.gradle.JibPlugin;
+import com.google.cloud.tools.jib.gradle.TestProject;
+import com.google.cloud.tools.jib.plugins.common.SkaffoldSyncMapTemplate;
+import com.google.cloud.tools.jib.plugins.common.SkaffoldSyncMapTemplate.FileTemplate;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.BuildTask;
+import org.gradle.testkit.runner.TaskOutcome;
+import org.gradle.testkit.runner.UnexpectedBuildFailure;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+/** Tests for {@link FilesTask}. */
+public class SyncMapTaskTest {
+
+  @ClassRule public static final TestProject simpleTestProject = new TestProject("simple");
+  @ClassRule public static final TestProject multiTestProject = new TestProject("multi-service");
+  @ClassRule public static final TestProject warProject = new TestProject("war_servlet25");
+
+  /**
+   * Verifies that the files task succeeded and returns the list of paths it prints out.
+   *
+   * @param project the project to run the task on
+   * @param moduleName the name of the sub-project, or {@code null} if no sub-project
+   * @return the list of paths printed by the task
+   */
```

Comment:
Missing a `@params` for `params`, and a `@throws` for `IOException` (not that important since it's private though)